### DISCUSSION TITLE
Cannot log in using 21Vianet

### DISCUSSION THREAD

**Discussion by SEUZTh** [abraunegg/onedrive#2151] [Q&A] [answered]
I want to use Office 365 operated by 21Vianet in China on Ubuntu 20.04.
But the software always jumps to the international version login interface, not the Chinese version.

**Comment by abraunegg** [comment, marked as answer]
@SEUZTh 
To use 21Vianet you must follow https://github.com/abraunegg/onedrive/blob/master/docs/national-cloud-deployments.md#how-to-configure-access-to-specific-microsoft-azure-deployments

**Reply by SEUZTh** [reply]
![image](https://user-images.githubusercontent.com/50619570/192129915-0db2a1ff-086e-493b-bee7-7d26a6d1a679.png)
Because Azure is not provided by 21Vianet, my education mailbox cannot be logged in.

**Comment by abraunegg** [comment]
@SEUZTh 
I am sorry but I cant read Chinese.

Your potential options to solve:

1. Use only `azure_ad_endpoint = "CN"` in your 'config' file, using the default settings
2. Speak to your education provider to assist getting the client configured correctly.

Unfortunatly there is nothing else here that I can help with.

**Reply by SEUZTh** [reply]
Thanks very much!
I found the website "https://portal.azure.cn/#home" and your software worked.

**Reply by abraunegg** [reply]
@SEUZTh 
Thanks for the feedback. Documentation has been updated via https://github.com/abraunegg/onedrive/pull/2152 to reflect to use the specific .cn portal for 21Vianet
